src/Filtering/BinaryMathematicalMorphology/ThinImage/Code.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python
+
+import sys
+import itk
+
+PixelType = itk.UC
+Dimension = 2
+ImageType = itk.Image[PixelType, Dimension]
+
+if len(sys.argv) == 2:
+    image = itk.imread(sys.argv[1])
+
+else:
+    # Create an image
+    start = itk.Index[Dimension]()
+    start.Fill(0)
+
+    size = itk.Size[Dimension]()
+    size.Fill(100)
+
+    region = itk.ImageRegion[Dimension]()
+    region.SetIndex(start)
+    region.SetSize(size)
+
+    image = ImageType.New(Regions=region)
+    image.Allocate()
+    image.FillBuffer(0)
+
+    # Draw a 5 pixel wide line
+    for i in range(20, 80):
+        for j in range(50, 55):
+            index = itk.Index[Dimension]()
+            index[0] = i
+            index[1] = j
+            image.SetPixel(index, 255)
```

Comment:
We can reduce this to:

```
image[20:80, 50:55] = 255
```

Reply:
I had to swap the 20:80 and 50:55 to get the line going the correct way, but then it worked, thank you.